src/main/java/ch/njol/skript/expressions/ExprSaturation.java

```diff
@@ -39,19 +36,29 @@
 import ch.njol.skript.lang.Expression;
 import ch.njol.skript.lang.ExpressionType;
 import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.util.Getter;
 import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
 
 /**
  * Used to set saturation of players. Number is used in case something changes in future...
  * @author bensku
+ * 
+ * Cleaned up by LimeGlass (2.2-dev35)
  */
 @Name("Saturation")
-@Description("The saturation of a player.")
-@Examples({"saturation of player is 20 #Not hungry!"})
-@Since("2.2-Fixes-V10")
+@Description("The saturation of the player(s).")
+@Examples("set saturation of player to 20 #Full hunger")
+@Since("2.2-Fixes-v10, 2.2-dev35")
```

Comment:
What you added in this version?

Reply:
Changed to SimplePropertyExpression but people hate that apparently as expressed by users in above conversations.